### DISCUSSION TITLE
Add support for ignoring unused imports diagnostics using #noqa

### DISCUSSION THREAD

**Discussion by luabud** [microsoft/pylance-release#5080] [Enhancement]
This is related to the problem mentioned at https://github.com/microsoft/pylance-release/discussions/4902, regarding implicit imports being removed when running the "Remove unused imports" code action. This would allow lines marked with #noqa to be skipped from removal.
A lot of linters such as Pylint and Flake8 already support #noqa to ignore diagnostics for specific lines. It might make sense for Pylance to support it too, at least for unused import scenarios

**Comment by tl24** [comment]
AI to the rescue.  A google search recommended using an _ in the import and that made pylint happy as well as prevented it from being removed by pylance as well.

```
from .mocks import unstub_and_verify as _
```

**Reply by heejaechang** [reply]
our recommendation is `from .mocks import unstub_and_verify as unstub_and_verify`
